enctests/testframework/main.py

```diff
@@ -302,11 +302,11 @@ def vmaf_compare(source_clip, test_ref, testname, comparisontestinfo, source_pat
     log_file_object.flush() # Need to flush it to make sure its before the subprocess logging.
 
     env = os.environ
-    if 'LD_LIBRARY_PATH' in env:
-        env['LD_LIBRARY_PATH'] += f'{os.pathsep}{VMAF_LIB_DIR}'
+    # if 'LD_LIBRARY_PATH' in env:
+    #     env['LD_LIBRARY_PATH'] += f'{os.pathsep}{VMAF_LIB_DIR}'
 
-    else:
-        env.update({'LD_LIBRARY_PATH': VMAF_LIB_DIR})
+    # else:
+    #     env.update({'LD_LIBRARY_PATH': VMAF_LIB_DIR})
```

Comment:
Let's rely on git and just remove code rather than commenting it out in cases like this.

Reply:
Done